packages/next-international/src/i18n/create-use-i18n.ts

```diff
@@ -18,26 +10,25 @@ export function createUsei18n<Locale extends BaseLocale>(I18nContext: Context<Lo
       throw new Error('`useI18n` must be used inside `I18nProvider`');
     }
 
-    function createT<Scope extends Scopes<Locale> | undefined>(scope: Scope) {
-      function t<Key extends LocaleKeys<Locale, Scope>, Value extends LocaleValue = ScopedValue<Locale, Scope, Key>>(
-        key: Key,
-        ...params: Params<Value>['length'] extends 0 ? [] : [ParamsObject<Value>]
-      ): string;
-      function t<Key extends LocaleKeys<Locale, Scope>, Value extends LocaleValue = ScopedValue<Locale, Scope, Key>>(
-        key: Key,
-        ...params: Params<Value>['length'] extends 0 ? [] : [ReactParamsObject<Value>]
-      ): React.ReactNode;
-      function t<Key extends LocaleKeys<Locale, Scope>, Value extends LocaleValue = ScopedValue<Locale, Scope, Key>>(
+    function createT() {
```

Comment:
I wonder if we could keep the current behavior of having a single `createT` function, that takes the `scope` or `undefined` to build both `useI18n` and `useScopedI18n` functions, instead of duplicating most of the code. What do you think?